src/main/java/org/alliancegenome/curation_api/services/associations/alleleAssociations/AlleleGeneAssociationService.java

```diff
@@ -60,8 +60,6 @@ public ObjectResponse<AlleleGeneAssociation> upsert(AlleleGeneAssociation uiEnti
 			return null;
 		}
 		dbEntity = alleleGeneAssociationDAO.persist(dbEntity);
-		addAssociationToAllele(dbEntity);
```

Comment:
Failing integration tests suggest that this may be needed (or something needs to replace it).  As far as I know, we haven't yet implemented CRUD operations for the other associations types changed here.

Reply:
Also, I'm not convinced our integration tests are sufficient to catch problems with updates in the bulk loads.  For example. the update test for AgmAlleleAssociations only checks that internal/obsolete are updated but doesn't try to update values for zygosity / created_by / updated_by / date_created / date_updated.

Reply:
I guess I might not understand here but here is the test:

```java
private String reference = "AGRKB:000000001";
private String evidenceCodeCurie = "DATEST:Evidence0001";

body("entity.evidence[0].curie", is(reference)).
body("entity.evidenceCode.curie", is(evidenceCodeCurie)).
```

However this is what is in the file:
```java
    "evidence_code_curie": "DATEST:Evidence0001",
    "evidence_curies": [
      "PMID:25920550"
    ],
```

Reply:
No, they're references created in earlier integration tests.  AGRKB:000000001 = PMID:25920550 and AGRKB:000000021 = PMID:25920551.  In bulk loads we're expecting the xref to be submitted rather than the AGR curie.  That test failing suggests that the update isn't working.

Reply:
Never mind the association was not getting updated.... because of the early added return statement.